### ISSUE TITLE
Cannot open the camera

### ISSUE THREAD

**Issue by multiflexi** [AravisProject/aravis#167]
Hi,
I am testing infrared camera Tachyon 16k from NIT. I tried latest Aravis release and also master branch but without success. I am attaching output of arv-test and camera features and screenshots or viewer and QArv. I was able to acquire image in 3rd party viewer. I think the issue is that Aravis does not figure out the resolution which is 128x128 and shows just 0 in arv-test. Any ideas?

[arv-test.log](https://github.com/AravisProject/aravis/files/2057870/arv-test.log)
[cam_features.txt](https://github.com/AravisProject/aravis/files/2057871/cam_features.txt)
![screenshot from 2018-05-31 12-18-02](https://user-images.githubusercontent.com/11663844/40778431-6b9dc9c6-64d1-11e8-8a99-a5adbbb14a04.png)
![screenshot from 2018-05-31 12-18-40](https://user-images.githubusercontent.com/11663844/40778432-6bbbdcf4-64d1-11e8-9d88-7e8dab422608.png)

**Comment by EmmanuelP** [comment]
Please attach the output of `arv-camera-test -d all:3`.

**Comment by multiflexi** [comment]
I let it run for 30 min, I hope it is enough.
[arv-camera-test.log](https://github.com/AravisProject/aravis/files/2067362/arv-camera-test.log)

**Comment by multiflexi** [comment]
Is the issue evident from the log?

**Comment by EmmanuelP** [comment]
Nope. The region seems to be correctly returned by arv-camera-test.
Could you attach the output of:
```
arv-tool-0.6 control SensorHeight SensorWidth Width Height -d device:3,evaluator:3
```

**Comment by multiflexi** [comment]
[arv-tool.log](https://github.com/AravisProject/aravis/files/2075260/arv-tool.log)
Here you go.

**Comment by EmmanuelP** [comment]
And the output of: `arv-tool-0.6 features` ?

**Comment by multiflexi** [comment]
[arv-tool-0.6.log](https://github.com/AravisProject/aravis/files/2076336/arv-tool-0.6.log)

**Comment by multiflexi** [comment]
I am now investigating what is wrong. Does Aravis require variables SensorWidth and SensorHeight in the XML file? According to Genicam SFNC they are recommended but not mandatory. When I compare the XML from camera and simulator which works, this is the difference I see: the camera  XML is missing those 2 variables and has only WidthMax and HeightMax.

**Comment by EmmanuelP** [comment]
Thanks for investigating this issue.
SensorHeight and SensorWidth are not used by the viewer.
I would recommend to try to debug with tests/arv-camera-test first. It is much more simpler to understand than arv-viewer with all that UI related code.
As I said previously, my first bet would be a non standard acquisition setup sequence. In order to set a fixed framerate acquisition, a number of features must be set, and this is most of the time vendor dependent. Have a look at the camera documentation.

**Comment by EmmanuelP** [comment]
It is a bit strange there is no TriggerMode and TriggerSelector features.
